internal/resources/resource_host.go

```diff
@@ -44,6 +44,7 @@ const (
 	hPortalCommOkay       = "portal_comm_okay"
 	hPwrState             = "power_state"
 	hLabels               = "labels"
+	hISCSIDiscoveryIP     = "iscsi_discovery_ip"
```

Comment:
Please use plural since it is a list:
   hISCSIDiscoveryIPs    = "iscsi_discovery_ips"

Reply:
No longer a list so back to just 1 address